CONTRIBUTING.md

```diff
@@ -21,6 +21,7 @@ If you don't have time to contribute, that's fine. There are other easy ways to
   - 📋 [Improving the Documentation](#-improving-the-documentation)
   - 🌐 [Improving the website](#-improving-the-website)
   - 🎨 [Improving the Design](#-improving-the-design)
+  - 🚀 [Custom Markdown Writing Style Guide](./md-style-guide.md)
```

Comment:
This link to open with proper render should be https://json-schema.org/md-style-guide

In addition I think this link doesn't go as part of the list of types of contribution, instead I see this as a section H2 section before Triage with a title "Markdown Style Guide" and a short paragraph to briefly introduce this guide for users contributing to docs or other types of written content like blogs or case studies.

Reply:
We also need to consider how this page will be reachable from the website.

Reply:
> This link to open with proper render should be https://json-schema.org/md-style-guide
> 
> In addition I think this link doesn't go as part of the list of types of contribution, instead I see this as a section H2 section before Triage with a title "Markdown Style Guide" and a short paragraph to briefly introduce this guide for users contributing to docs or other types of written content like blogs or case studies.

sure would be happy to do these changes.